specs/agents/tracing-instrumentation-aws.md

```diff
@@ -44,7 +44,15 @@ The format should be `DynamoDB <ActionName> <TableName>`. So for example, `Dynam
 ### SQS (Simple Queue Service)
 
 AWS Simple Queue Service is a message queuing service. The [messaging spec](tracing-instrumentation-messaging.md) can 
-be used for instrumenting SQS, but the follow specifications supersede those of the messaging spec.
+be used for instrumenting SQS. The follow specifications supersede those of the messaging spec:
+
+For a batch send message operation, the span name is `SQS SEND_BATCH to MyQueue`. The `span.action` is `send_batch`.
```

Comment:
@alex-fedotyev Do you agree the user would want to know if something was a single message send or batch message send? Same question below for the delete operation.